app/plot/plotter.py

```diff
@@ -442,9 +527,16 @@ def create_plot(self):
             plot.width_policy = "max"
 
             # Add a label in the center
+            highlight_category = self.config["filters_defaults"][
+                "single_choice_highlight"
+            ]
```

Comment:
This was already defined before and is not depending on the code of the config. Or there is something missing there...